packages/api/docs/present/models/records.yaml

```diff
@@ -0,0 +1,91 @@
+record:
```

Comment:
I don't think this entirely matches the model defined here https://github.com/PermanentOrg/stela/blob/c3beef495a190855ed3e4ea7162a09796b5db333/packages/api/src/record/models.ts#L1